elks/include/linuxmt/mm.h

```diff
@@ -54,10 +54,9 @@ extern void pokew(__u16,__u16,__u16);
 extern __u8  peekb(__u16,__u16);
 extern __u16 peekw(__u16,__u16);
 
-extern void fmemcpy(__u16,__u16,__u16,__u16,__u16);
-extern void fmemset(char *,__u16,__u16,size_t);
-
-extern void far_memmove(unsigned,unsigned,unsigned,unsigned,unsigned);
+/* memory primitives - count in bytes */
+extern void fmemcpy (word_t dst_off, seg_t dst_seg, word_t src_off, seg_t src_seg, word_t count);
```

Comment:
I like seeing these header files changed to have the variable names in them instead of a bunch of nameless type specifications.

Reply:
You are welcome.